src/Files.App/Resources/PreviewPanePropertiesInformation.json

```diff
@@ -467,6 +467,18 @@
 		},
 		"ID": null
 	},
+	{
+		"NameResource": "GitOriginRepositoryName",
+		"SectionResource": "PropertySectionCore",
+		"IsReadOnly": true,
+		"ID": null
+	},
+	{
+		"NameResource": "GitCurrentBranch",
+		"SectionResource": "PropertySectionCore",
+		"IsReadOnly": true,
+		"ID": null
+	},
```

Comment:
Because the tags field is often blank, it might look better to put these after tags. Alternatively we can hide the tags field when there aren't any tags.

Reply:
Hiding tags when there are none is a good option.

Alternatively you could have an Add Tag button here, possibly with text saying "This file/folder currently has no tags attached"

Reply:
As "tags" is pluralised, maybe this icon is more appropriate...
![image](https://github.com/files-community/Files/assets/7389110/0d9cefb4-4e29-4d5c-b730-4e4e26760706)

Reply:
The "Add Tag" is a pretty good idea @mdtauk, I'd suggest opening a feature request for that.

Reply:
done #12821

Reply:
> Because the tags field is often blank, it might look better to put these after tags. Alternatively we can hide the tags field when there aren't any tags.

Tags and git info switched.